### PR TITLE
Milestone: validations and model specs

### PR DESCRIPTION
**Pull request by Roseokpe**
In this project, I added validations to my models and created specs for them.

### CODE REVIEW

File: app/models/post.rb [hunk 1]
```diff
@@ -5,6 +5,10 @@ class Post < ApplicationRecord
 
   after_save :update_posts_counter
 
+  validates :title, presence: true, length: { maximum: 250 }
+  validates :comments_counter, comparison: { greater_than: 0 }
```
Comment (Ismailco) [inline app/models/post.rb]: - [ ] CommentsCounter must be an integer greater than or equal to zero, not just greater than zero, kindly implement this so the comment counter can accept also a zero value

File: app/models/post.rb [hunk 2]
```diff
@@ -5,6 +5,10 @@ class Post < ApplicationRecord
...
+  validates :likes_counter, comparison: { greater_than: 0 }
```
Comment (Ismailco) [inline app/models/post.rb]: - [ ] LikesCounter must be an integer greater than or equal to zero, not just greater than zero, kindly implement this so the likes counter can accept also a zero value

File: spec/models/post_spec.rb [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+      expect(subject).to_not be_valid
+    end
+
+    it 'comments_counter should be present' do
+      subject.comments_counter = nil
+      expect(subject).to_not be_valid
+    end
+
+    it 'likes_counter should be present' do
+      subject.likes_counter = nil
```
Comment (Ismailco) [inline spec/models/post_spec.rb]: - [ ] [OPTIONAL] For comments and likes counters, you can include testing if the value is less than zero, or the value is zero... this will test the validations in the models

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @Roseokpe, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Deepakdanger — APPROVED** [review]
Hi @Roseokpe, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/models/post_spec.rb (hunk 3 above)
Comment (Roseokpe) [reply spec/models/post_spec.rb]: Thank you
